src/exampleTypes/TermsStatsTable.js

```diff
@@ -1,12 +1,83 @@
+import _ from 'lodash/fp'
 import React from 'react'
 import { observer } from 'mobx-react'
 import { exampleTypes } from 'contexture-client'
 import injectTreeNode from '../utils/injectTreeNode'
-import ExpandableTable from '../layout/ExpandableTable'
+import ExpandableTable, { Column } from '../layout/ExpandableTable'
+import { applyDefaults, inferSchema } from '../utils/schema'
 
 let TermsStatsTable = injectTreeNode(
-  observer(({ node, ...props }) => (
-    <ExpandableTable {...props} data={node.context.terms} />
+  observer(({ infer, fields, node, criteria, tree, children, ...props }) => (
+    <div>
+      <div>
+        Filter:
+        <input
+          type="text"
+          value={node.filter}
+          onChange={async e => {
+            await tree.mutate(node.path, { filter: e.target.value })
+            // TODO: This seems like a contexture-client issue. We shouldn't need this line:
+            await tree.refresh(node.path)
```

Comment:
It is a missing reactor: https://github.com/smartprocure/contexture-client/blob/master/src/exampleTypes.js#L186

Just add a self reactor for `filter` there

Reply:
you can also use tree.lens + futil.domLens.value:

```js
<input
  type="text"
  {...F.domLens.value(tree.lens(node.path, 'filter'))}
/>```